devices.json

```diff
@@ -3000,6 +3000,28 @@
          "vendor_google_taimen"
       ]
    },
+   {
+      "name": "Mi CC9 Pro/Note 10",
+      "brand": "Xiaomi",
+      "codename": "tucana",
+      "supported_versions": [
+         {
+            "version_code": "eleven",
+            "stable": false
+         },
+         {
+            "version_code": "eleven_plus",
+            "stable": false
+         }
+      ],
+      "repositories": [
```

Comment:
Will you need the firmware repo some time in the future too? Might as well add it now.

Reply:
Not that soon at least. But down the road it might be turning into a thing same as back in the day with dipper. Also depending of how sweet is going to do about it as both me and Zidan will share the common-tree repo.